src/time-slot-week-view/time-slot-week.styles.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import { TextStyleHelper } from "../text";
+
+export const ColumnWeekCell = styled.div`
+    grid-column: 1 / -1;
+    display: flex;
+    min-height: 7.625rem;
+`;
+export const TimeSlotText = styled.div`
+    ${TextStyleHelper.getTextStyle("XSmall", 600)}
+    margin: 1rem 0rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    line-height: 16px;
+    letter-spacing: 0.12px;
```

Comment:
this has no effect. anyway based on Figma, XSmall semibold should be sufficient

missing the N1 color